state/mysql/mysql.go

```diff
@@ -242,7 +242,7 @@ func (m *MySQL) ensureStateTable(stateTableName string) error {
 		// in on inserts and updates and is used for Optimistic Concurrency
 		createTable := fmt.Sprintf(`CREATE TABLE %s (
 			id varchar(255) NOT NULL PRIMARY KEY,
-			value json NOT NULL,
+			value text NOT NULL,
```

Comment:
For my understanding, what breaks when this column is set to `json`?

Reply:
pls. check https://github.com/dapr/components-contrib/pull/1118#discussion_r704013615